test-app/package.json

```diff
@@ -8,6 +8,7 @@
     "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
+    "lint:types": "glint",
```

Comment:
does this run in CI?

Reply:
I don't actually see any linting going on in CI -- can you verify that linting and type checking is happening in the workflow files?

thanks!

Reply:
I believe the lint scripts are actually broken on main (at least on my Windows comp) #19

I added `pnpm run lint` to the test app CI job, which should trigger a `lint:types` run